jbehave-support-core/docs/Rest-api.md

```diff
@@ -144,3 +144,29 @@ When [POST] request to [TEST]/[user/] is sent with data:
 | @header.Content-Type | application/json;charset=utf-8 |
 | @body                | Anything you want to send.     |
 ```
+
+#### Handling JSON data types
+All JSON data will be send as a string, unless specified otherwise. To specify a data type, use optional `type` column in your data table. These data types are supported:
+ + `string`
+ + `boolean`
+ + `number`
+ + `null` (_only through the `{NULL}` command_)
```

Comment:
I would probably remove this  since its written above to use it in the type column (at least for me it seems confusing 🙂 ). The mention of null below plus the example should hopefully be enough.